### ISSUE TITLE
using usePopup=true results in the loss of the request settings

### ISSUE THREAD

**Issue by makeoverweb** [patelmayankce/react-apple-login#28]
Hello, tell me why when using usePopup in the request parameters on https://appleid.apple.com/appleauth/auth/oauth/authorize is form_post replaced by web_message?And the scope is also lost.And if you specify usePopup=false, then everything is fine.
But this is inconvenient, I would like to perform authorization using usePopup.

**Comment by vaishaliplms** [comment]
same issue at my end

**Comment by harizyusoff** [comment]
@makeoverweb any update / your implementation on this ?

**Comment by makeoverweb** [comment]
@harizyusoff No, I just stopped using usePopap. Although it may not be so convenient for the user((

**Comment by harizyusoff** [comment]
Okay got it @makeoverweb but how did you redirect back to your website after it is successfully login from redirectURI ?

**Comment by makeoverweb** [comment]
@harizyusoff  I used to think that I needed to redirect the user myself after successful authorization in the apple system.But redirectUri is indicated for this, in order to automatically redirect the user to this page. As soon as you enter your data, they are correct, click continue and at this moment it seems that a request for a backend (I did not go into the implementation) goes away on which any actions necessary for authorization can occur (for example, installing a token in cookies).And that's it, then redirect to a page that already has authorization
